docs/src/understand/sft.rst

```diff
@@ -70,94 +74,674 @@ Prerequisites
+
+In the case of a conference call the client starting the conference transmits the `TURN` servers and credentials to the `SFT` server as `SFT` servers do not have their own connection to the backend.
+
+Encoding
+........
+
+The codec used for streaming is `Opus <https://en.wikipedia.org/wiki/Opus_(audio_format)>`__ for audio and `VP8 <https://en.wikipedia.org/wiki/VP8>`__ for video.
+
+Opus can use variable bit rate encoding (`VBR <https://en.wikipedia.org/wiki/Variable_bitrate>`__) or constant bitrate encoding (`CBR <https://en.wikipedia.org/wiki/Constant_bitrate>`__).
+
+Users can choose to enforce `CBR` in one-to-one calls in the settings.
+
+Conference calls always use `CBR` encoding.
+
+In custom builds of Wire it is possible to enforce the `CBR` option for one-to-one calls, too and remove this option for the users.
+
+`CBR` has the advantage of eliminating potentially undesired information about packet length but might have an impact on call quality on slow networks.
+
+It is sufficient if one of the two parties of a call enables the `CBR` option, `CBR` will then always be used for calls of that user.
+
+When `CBR` is used, the calling screen will display `CONSTANT BIT RATE`.
+
+In video calls the CBR option affects the audio streams like in audio calls, but the calling screen will not display `CONSTANT BIT RATE`.
+
+One-on-One calls
+~~~~~~~~~~~~~~~~
+
+Call setup example
+..................
+
+The following is an example for setting up a one-to-one call with client `A` calling client `B`.
+
+Client `A` connects to `TURN` server A and client `B` to `TURN` server B.
+
+In practice these two `TURN` servers could be the same server.
+
+The separation was chosen to reflect the fact that the external side of the `TURN` servers connects via `UDP`.
+
+Clients may also directly connect via UDP to either other clients that are directly reachable or to a `TURN` server that a client is connected to.
+
+.. figure:: img/sft-call-setup-example.png
+   :alt: Call setup example
+   :align: center
+
+   client `A` connecting with client `B` via `TURN` server A and `TURN` server B
+
+Before a call can be set up, clients need to receive a call configuration from their associated backend.
+
+This configuration is received when clients come online after they were offline for a longer time and it is frequently refreshed while being online.
+
+The refresh interval (TTL, `Time To Live <https://en.wikipedia.org/wiki/Time_to_live>`__) can be set on the backend and is transmitted to clients in the configuration.
+
+The configuration contains all available `TURN` servers, credentials to connect to the `TURN` server, and all available transport protocols.
+
+`TURN` servers can be configured to allow any combination out of `UDP`, `TCP`, and `TLS`.
+
+They are listening on the following ports:
+
+* `3478` for `UDP`
+* `3478` for `TCP`
+* `5349` for `TLS`
```

Comment:
This should be "TCP/TLS" or "TCP with TLS"

Reply:
I'm trying to double check this, it may be offering dTLS over UDP as well. Unsure

Reply:
@e-lisa are you aware of this page: https://wearezeta.atlassian.net/wiki/spaces/BAC/pages/995491856/sftd+and+coturn+ports

(You should likely not need to research these ports. I did :smile: )